pay-api/src/pay_api/resources/v1/account.py

```diff
@@ -78,6 +78,29 @@ def get_account(account_number: str):
     return jsonify(response), status
 
 
+@bp.route('/<string:account_number>/eft', methods=['PATCH'])
+@cross_origin(origins='*')
+@_tracing.trace()
+@_jwt.has_one_of_roles([Role.SYSTEM.value])
+def patch_account(account_number: str):
+    """Enable eft for an account."""
+    current_app.logger.info('<patch_account')
+
+    # Check if user is authorized to perform this action
+    check_auth(business_identifier=None, account_id=account_number, one_of_roles=[EDIT_ROLE])
+    try:
+        response = PaymentAccountService.enable_eft(account_number)
+    except ServiceUnavailableException as exception:
+        return exception.response()
+
+    status = HTTPStatus.ACCEPTED \
```

Comment:
Why would this be ACCEPTED or OK?

Shouldn't it throw a 400 if the org doesn't exist?